jdk-tool-plugin/src/main/resources/org/jenkinsci/plugins/jdk_tool/Messages.properties

```diff
@@ -0,0 +1,28 @@
+# The MIT License
+#
+# Copyright (c) 2004-2009, Sun Microsystems, Inc., Seiji Sogabe
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+JDKInstaller.FailedToInstallJDK=Failed to install JDK. Exit code={0}
+JDKInstaller.RequireOracleAccount=Installing JDK requires Oracle account. <a href="{0}" target="_new">Please enter your username/password</a>
+JDKInstaller.UnableToInstallUntilLicenseAccepted=Unable to auto-install JDK until the license is accepted.
+JDKInstaller.DescriptorImpl.displayName=Install from java.sun.com
+JDKInstaller.DescriptorImpl.doCheckId=Define JDK ID
+JDKInstaller.DescriptorImpl.doCheckAcceptLicense=You must agree to the license to download the JDK.
```

Comment:
trailing newline

Reply:
Fixed in [d62e8dc](https://github.com/jenkinsci/jenkins/pull/3301/commits/d62e8dc0ce90f001bcc3d5b4bf05a64415a01d11#diff-c4c99b563e6a2043e20958e57d73823aR28).